src/sgame/sg_bot_util.cpp

```diff
@@ -1924,18 +1924,38 @@ void BotFireWeaponAI( gentity_t *self )
 			break;
 		case WP_ALEVEL3_UPG:
 		{
+			glm::vec3 origin = VEC2GLM( self->s.origin );
```

Comment:
These can be moved closer to the point of use

Reply:
Updated